public/data-retention/thank-you.html

```diff
@@ -19,7 +19,7 @@
       s.parentNode.insertBefore(ga, s);
     })();
   </script>
-  <link href="/index.css" rel="stylesheet">
+  <link href="../index.css" rel="stylesheet">
```

Comment:
You don't need `..` for `.css` and `img` stuff, so just use `/` (root) directory for those.